app/views/base_mailer/access_details_email.html.slim

```diff
@@ -2,32 +2,33 @@ doctype html
 html
   head
     meta content='text/html; charset=UTF-8' http-equiv='Content-Type'
+    meta name="viewport" content="width=device-width"
   body
-    = render partial: 'base_mailer/shared/gov.uk.logo'
+    table width="100%" cellpadding="0" cellspacing=0
+      tr
+        = render partial: 'base_mailer/shared/gov.uk.logo'
 
-    table style="padding-left:60px;"
       tr
+        td width="8%"
+          p
+            / leave empty
```

Comment:
making sure the visual margin on the left is always there. I can remove the comment - but the empty `p` should be there. Just having an empty `td` is not enough to be confident it will be rendered correctly.